flink-connector/src/test/resources/flink-tests/hive-site.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-stylesheet type="text/xsl" href="configuration.xsl"?>
+<!--
+  ~ Copyright 2024 Datastrato Pvt Ltd.
+  ~ This software is licensed under the Apache License version 2.
+  -->
+
+<!-- Put site-specific property overrides in this file. -->
+
+<configuration>
```

Comment:
could these properties managed by Gravitino?

Reply:
@FANNG1 I think it can be managed by Gravitino.  At first, I want to keep the options consistent with the Flink. In Flink, it will use `hive-conf-dir` and `hadoop-conf-dir`. In Gravitino, we have the GravitinoHiveCatalog, we can override them.

Reply:
I'm a little confusing about the current catalog properties convert logic. when converting flink properties to Gravitino catalog properties, all configurations load from config file will be add "flink.bypass." prefix which means the configuration properties are managed by Gravitino not the configuration file path like `hive-conf-dir` and `hadoop-conf-dir`.

Reply:
@FANNG1 I think we should store `flink.bypass.hive-conf-dir` and `flink,bypass.hadoop-conf-dir` in Gravitino. Because when using Flink SQL to create the HiveCatalog, the `HiveCatalogFactory` will validate the properties. They must be `hive-conf-dir` and `hadoop-conf-dir`. But we should store the value with the prefix `flink.bypass.`. 

Because I want to keep the consistency with the flink logic, so I use the `requiredOptions` from the HiveCatalogFactory in flink-connector-hive, not create a new `requiredOptions`. If we create new `requiredOptions` that already have the prefix, we will not need to add the prefix anymore. What do you think?

Reply:
@FANNG1 The HiveCatalogFactory will validate the options. If there are some options that not supported by HiveCatalogFactory, it will be failed.
```
final FactoryUtil.CatalogFactoryHelper helper =
        FactoryUtil.createCatalogFactoryHelper(this, context);
    helper.validate();
```